Source/SelfService/Web/logging/logFilter/filterSelect.tsx

```diff
@@ -2,8 +2,35 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import React from 'react';
+
+import { themeDark } from '../../theme/theme';
 import { Select, SelectProps } from '@mui/material';
 
+const styles = {
+    'padding': '5px 14px',
+    'borderRadius': '4px',
+    'transition': 'all .3s',
```

Comment:
I like that you are leaning on the transition to mimic what the default text button does. It would be good to lean on the same values as the button, though, and not have to maintain this specific transition. In our call earlier today, you found the transition duration and easing variables in MUI (https://mui.com/material-ui/customization/transitions/#durations), so I'm leaving this as a reference

```suggestion
    'transition': (theme: any) => theme.transitions.create(['all'], { duration: theme.transitions.duration.short }),
```